src/state/actions/dispatchArNSUpdate.ts

```diff
@@ -52,7 +52,6 @@ export async function dispatchArNSUpdate({
 
     // TODO: move all of these async calls to unique hooks and remove the need for the dispatchArNSUpdate action
     const arioContract = ARIO.init({
-      hyperbeamUrl,
```

Comment:
this was added so the ANTRegistry internally in the sdk uses the hyperbeam ACL - why does it need to be removed?